roles/controller_workflow_job_templates/tasks/main.yml

```
@@ -1,4 +1,15 @@
 ---
+# Create links between workflow node
+- name: Loop over nodes in schema to add to workflow templates
+  ansible.builtin.include_tasks: add_workflows_schema.yml
+  loop: "{{ controller_workflows | default(workflow_job_templates) }}"
+  loop_control:
+    loop_var: __workflow_loop_item
+    label: "Workflow {{ __workflow_loop_item.name }}"
+  when:
+    - (__workflow_loop_item.state | default(platform_state | default('present'))) is not match('present')
```

I think this should be then it's absent rather than when it's 'not present'.

```suggestion
    - (__workflow_loop_item.state | default(platform_state | default('present'))) is match('absent')
```